src/Sylius/Bundle/AdminBundle/Console/Command/ListAdminUsersCommand.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Console\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'sylius:admin-user:list',
+    description: 'Displays a list of all admin users along with their details.',
+)]
+final class ListAdminUsersCommand extends Command
+{
+    protected SymfonyStyle $io;
```

Comment:
Since class is final and closed to extension.

```suggestion
    private SymfonyStyle $io;
```